ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-release-1.1.yaml

```diff
@@ -0,0 +1,152 @@
+base_images:
+  base:
+    name: "4.12"
```

Comment:
ince external-dns isn't tied to an OpenShift release, it is confusing to use or reference OpenShift or OpenShift versions. Can you use a different name, namespace, tag, builder etc?  I expect to not see "4.12", "ocp", or "openshift" anywhere in here.

Reply:
I don't see a big problem in using the OCP image here. It'll be used as the base (Docker `FROM`) image for the operator.  I think of  it as a sort of minimal Kube version in combination with RHEL which covers the most of the future customers of this operator.

Reply:
That sounds reasonable.  Could you please get someone from the release team to validate that you have what is needed here, in order to generate updates for releases after 4.12? If there are OpenShift CI updates after 4.12, this config should be using them, and the things I suggested so far that I know about, you have not agreed would work.  So probably we need an expert on release CI generation files.

Reply:
Found `ubi-minimal` imagestream. Should be enough for external dns repositories.